README.md

```diff
@@ -32,6 +32,6 @@ would be a nice to do thing.
 2. Best practices. Design/Styling/CSS is not critical; functionality is.
 3. Define rules for validation.
 4. Use of upcoming JS frameworks like Vue/React
-5. Use Docker / ASP.NET Core./Web API
+5. Use Docker / ASP.NET Core / Web API
```

Comment:
please add more details

Reply:
Hey @A4Akash, I think this is sufficient for now. Please proceed😊